.gitignore

```diff
@@ -1,3 +1,8 @@
 /coverage/
 /vendor/
 /clover.xml
+composer.lock
+phpunit.xml
+composer.lock
+Time.php.xml
```

Comment:
what is `Time.php.xml` and `index.xml`??

Reply:
They get added by code coverage, didn't mean to add them